src/libtiled/map.h

```diff
@@ -411,6 +411,11 @@ class TILEDSHARED_EXPORT Map : public Object
      */
     QSet<SharedTileset> usedTilesets() const;
 
+    /**
+     * Returns a list of MapObjects to be updated by syncing objectTemplate
+     */
+    QList<MapObject*> syncObjectTemplate(const ObjectTemplate *objectTemplate);
```

Comment:
Function should not be necessary, this is already implemented as `MapDocument::updateTemplateInstances`.

Reply:
updateTileInstances asserts that the before- and after-template instances are different.

Reply:
Nope, I was wrong, that was just dumb. I don't know how I missed that.